doc/installationguide.md

```diff
@@ -211,6 +226,10 @@ used for the same purpose. For instance:
     the IoTAgent runs in a single thread. For more details about multi-core functionality, please refer to the
     [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and
     [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
+-  **fallbackTenant** - For Linked Data Context Brokers which do not support multi-tenancy, this provides an alternative mechanism for suppling the `NGSILD-Tenant` header.
+    Note that for backwards compatibility with NGSI v2, the `fiware-service` header is already used as alternative if the `NGSILD-Tenant` header is not supplied.
+-  **fallbackPath** - For Linked Data Context Brokers which do not support a service path, this provides an alternative mechanism for suppling the `NGSILD-Path` header.
+    Note that for backwards compatibility with NGSI v2, the `fiware-service-path` header is already used as alternative if the `NGSILD-Path` header is not supplied.
```

Comment:
Looking to ETSI GS CIM 009 V1.2.1 I haven't found references to NGSILD-Tenant or NGSILD-Path...

Is that correct or maybe I have missed it? If I'm correct, could you elaborate on this new two headers?

Reply:
New proposals for v1.3 - not decided yet.

Reply:
Thanks for the clarification. I think it would be a good idea to mention it (in a separate comment).

NTC